Source/WebKit/WebProcess/WebProcess.h

```diff
@@ -771,6 +779,11 @@ class WebProcess : public AuxiliaryProcess
 #endif
     bool m_hadMainFrameMainResourcePrivateRelayed { false };
     bool m_imageAnimationEnabled { true };
+    
+#if ENABLE(LOGD_BLOCKING_IN_WEBCONTENT)
+    Lock m_logDataLock;
+    Vector<LogData> m_logSet WTF_GUARDED_BY_LOCK(m_logDataLock);
```

Comment:
We probably don't want to use "Set" in the name. I find it a little confusing for a vector.